packages/syft/src/syft/service/code/user_code.py

```diff
@@ -507,16 +558,20 @@ def debox_asset(arg: Any) -> Any:
     return deboxed_arg, ArgumentType.REAL
 
 
-def syft_function_single_use(*args: Any, **kwargs: Any):
+def syft_function_single_use(
+    *args: Any, share_results_with_owners=False, **kwargs: Any
+):
     return syft_function(
         input_policy=ExactMatch(*args, **kwargs),
         output_policy=SingleExecutionExactOutput(),
+        share_results_with_owners=share_results_with_owners,
     )
 
 
 def syft_function(
     input_policy: Union[InputPolicy, UID],
     output_policy: Optional[Union[OutputPolicy, UID]] = None,
+    share_results_with_owners=False,
```

Comment:
Should we make it True by default because this is something that DS controls ?

Reply:
irina suggested False, I agree. In theory the output still could be data that is sensitive for the data owners so I would not have it on by default